packages/react/src/components/form/form.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  Asterisk,
+  Button,
+  Checkbox,
+  Field,
+  FieldLabel,
+  Form,
+  FormProps,
+  HelperText,
+  Input,
+  Radio,
+  Validation,
+} from '@onfido/castor-react';
+import React from 'react';
+import { Meta, Story } from '../../../../../docs';
+
+export default {
+  title: 'React/Form',
+  component: Form,
+} as Meta<FormProps<unknown>>;
+
+interface Values {
+  firstName: string;
+  lastName: string;
+  email: string;
+  day?: number;
+  month?: number;
+  year?: number;
+  source?: string;
+  agreement: boolean;
+}
+
+export const Playground: Story<FormProps<Values>> = (props) => (
+  <Form {...props}>
+    <Field>
+      <Input name="firstName" required>
+        First name <Asterisk aria-label="required" />
+      </Input>
+      <Validation state="error" for="valid">
+        Please fill in this field
+      </Validation>
+    </Field>
+
+    <Field>
+      <Input name="lastName" required>
+        Last name <Asterisk aria-label="required" />
+      </Input>
+      <Validation state="error" for="valid">
+        Please fill in this field
+      </Validation>
+    </Field>
+
+    <Field>
+      <Input type="email" name="email" required>
+        Email address <Asterisk aria-label="required" />
+      </Input>
+      <Validation state="error" for="valueMissing">
+        Please fill in this field
+      </Validation>
+      <Validation state="error" for="typeMismatch">
+        Please enter a valid email address
+      </Validation>
+    </Field>
+
+    <Field>
+      <FieldLabel htmlFor="dob-day">
+        Date of birth
+        <HelperText>DD / MM / YYYY</HelperText>
+      </FieldLabel>
+      <div
+        style={{
+          display: 'grid',
+          gap: '0.5rem',
+          grid: '1fr / 5rem 5rem 6rem',
+        }}
+      >
+        <Input
+          id="dob-day"
+          type="number"
+          name="day"
+          placeholder="DD"
+          min={1}
+          max={31}
+        />
+        <Input type="number" name="month" placeholder="MM" min={1} max={12} />
+        <Input
+          type="number"
+          name="year"
+          placeholder="YYYY"
+          min={new Date().getUTCFullYear() - 150}
+          max={new Date().getUTCFullYear()}
+        />
+      </div>
+    </Field>
+
+    <Field>
```

Comment:
Why not using Fieldset and FieldsetLegend here?

Reply:
I need the `Field` for `Validation`, don't really need the `Fieldset` but `FieldsetLegend` might be a good call.

Should we use `FieldsetLegend` without a `Fieldset`?

Using everything might be convoluted.

Reply:
Why don't you need a Fieldset?

I think this is what should surround grouped radios (and also checkboxes).

Also, as discussed on another thread, probably good to have "disabled" control on it.

Reply:
`Fieldset` already has `disabled`, `<fieldset>` supports it by default.

I don't need it because presumably a single `[name]` is just a single field, `<Radio>` is just an oddball.

Like I commented on the other thread, lt me research `<fieldset>` a bit more to see what makes more sense here.

Reply:
This looks good now, however why do you still need a parent Field?

Reply:
Hmm I was probably thinking about using `Validation` but that radio has no possible invalid state.
Will remove `Field` from there.